Task01/NarcissisticNumbers.py

```diff
@@ -0,0 +1,33 @@
+import os,sys
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+def main():
+    print_NarcissisticNumbers(1,1000)
```

Comment:
```suggestion
    print_NarcissisticNumbers(1, 1001)
```

range(n, m) - начало диапазона n, а конец m-1 -- (n, m]